src/pages/Cart.jsx

```diff
@@ -21,7 +22,10 @@ const Cart = () => {
           <CartProducts>
             {cartItems.length === 0 ? (
               <h1 className='h1'>
-                No items in the cart <br /> 🙈🙉🙊
+                No items in the cart <br /> 🙈🙉🙊 <br />
```

Comment:
I think Tanay mentioned that br tags shouldn't be used with JSX. I am not sure what the replacement for this is but you can consider looking into this.